spec/serializers/api/v1/user_serializer_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+describe Api::V1::OfferSerializer do
+
+  let(:donor) { create(:user) }
+  let(:donor_serializer) { Api::V1::UserSerializer.new(donor) }
+  let(:donor_json) { JSON.parse(donor_serializer.to_json) }
+  let(:admin) { create(:user, :reviewer) }
+
+  context "Donor" do
+    before { User.current_user = donor }
+    it "creates JSON" do
+      expect(donor_json['user']['id']).to eql(donor.id)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.